index.html

```diff
@@ -107,6 +107,74 @@ <h3>
       </dd>
     </dl>
   </section>
+  <section id="RTCVideoReceiverStats-dict*">
+    <h3>
+      <dfn>RTCVideoReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCVideoReceiverStats {
+      unsigned long numFreezes;
+      unsigned long numPauses;
+      double totalFreezesDuration;
+      double totalFramesDuration;
+      double sumOfSquaredFramesDuration;
+};</pre>
+    <dl data-link-for="RTCVideoReceiverStats" data-dfn-for="RTCVideoReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>numFreezes</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long">unsigned long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video freezes experienced by this receiver. It is a
+          freeze if frame duration, which is time interval between consecutively
+          rendered frames, is 3 times higher than the average frame duration.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>numPauses</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video pauses experienced by this receiver. Video is
+          considered to be paused if time passed since last received packet
+          exceeds 5 seconds.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>totalFreezesDuration</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total duration of video freezes, in seconds.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>totalFramesDuration</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total duration of rendered video frames, in seconds.
```

Comment:
What is the "duration of rendered video frames"? If we freeze, do we keep rendering the old frame? Or is this supposed to be some sort of total time minus freeze time? How is this calculated?

Reply:
Yes, we keep rendering old frame at freeze (we don't have anything to update it with).

How about this definition?:
Total duration of all rendered video frames, in seconds. This value is updated when a frame is rendered.

Reply:
Is this the time a frame is rendered for - so that if you have an 1-second freeze, the frame is displayed for 1 second, and this counter goes up by 1 second? Won't this be exactly the same as "time since first frame was rendered" - "time since last frame was rendered"?

Reply:
I'm wondering the same.